www.chuyu.me/es.xml

```diff
@@ -33,7 +33,7 @@
   <String Src="%WinOS%" Dst="WINOS Lite System"/>
   <String Src="%ChuyuTream%" Dst="&amp;nbsp;2017&amp;nbsp;equipo Chuyu"/>
   <String Src="%ICP%" Dst=""/>
-  <String Src="%Document%" Dst="Documentación"/>
+  <String Src="%Document%" Dst="Documentación de ayuda"/>
```

Comment:
Translation incorrect.

Correct is "Ayuda" o "Ver ayuda", not "Documentación de Ayuda".